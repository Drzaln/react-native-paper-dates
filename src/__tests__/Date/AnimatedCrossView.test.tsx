import { render } from '@testing-library/react-native'
import AnimatedCrossView from '../../Date/AnimatedCrossView'

it('renders AnimatedCrossView', () => {
  const { toJSON } = render(
    <AnimatedCrossView
      collapsed={false}
      calendar={<></>}
      calendarEdit={<></>}
    />
  )
  expect(toJSON()).toMatchSnapshot()
})
