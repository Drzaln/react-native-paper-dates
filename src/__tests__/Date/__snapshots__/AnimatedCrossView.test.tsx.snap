// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders collapsed AnimatedCrossView 1`] = `null`;
